packages/config/config/devices/0x032e/dat-101.json

```diff
@@ -0,0 +1,402 @@
+{
+	"manufacturer": "DEFARO",
+	"manufacturerId": "0x032e",
+	"label": "DAT-101",
+	"description": "Thermostat actuator DAT-101",
+	"devices": [
+		{
+			"productType": "0x0013",
+			"productId": "0x0021"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "State changes",
+			"description": "State changes -- general",
+			"maxNodes": 10,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "External temperature sensor",
+			"description": "A group of associations to connect external room temperature sensor",
+			"maxNodes": 1
+		}
+	},
+	"paramInformation": [
```

Comment:
Please read the [style guide](https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide) thoroughly and apply it here.

Reply:
This is not resolved, let me highlight a few examples.